style.css

```diff
@@ -73,3 +83,146 @@ body {
   gap: 2rem;
   align-items: center;
 }
+
+/* WORKS */
+.works {
+  background-color: #f7f7f7;
+  padding: 12.8rem 0 4.8rem 0;
+  margin-bottom: 4.8rem;
+}
```

Comment:
- [ ] Please use `display: grid` because this is a requirement for more detail check your [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). It is mandatory that you use both. Use `CSS Grid` in the main container that contains all of the cards and use `Flexbox` to lay out an individual card :+1: